Deeploy/Targets/Generic/Bindings.py

```
@@ -326,19 +327,35 @@
     for type in FloatDataTypes
 ]
 
-BasicConvTransposeBindings = [
+BasicConvTranspose1DBindings = [
     NodeBinding(
         ConvChecker(
             [PointerClass(type), PointerClass(type), PointerClass(type)],  # input, weight, bias
             [PointerClass(type)]),
-        ConvTransposeTemplate.referenceTemplate,
+        ConvTransposeTemplate.referenceTemplate1D,
         BasicTransformer) for type in FloatDataTypes
```

Ruff (0.15.20)

[error] Variable `type` is shadowing a Python builtin

(A001)